src/main/java/com/example/onlinebookstore/service/BookServiceImpl.java

```diff
@@ -34,4 +34,15 @@ public BookDto findById(Long id) {
                 .orElseThrow(() -> new EntityNotFoundException("Can't get book by id" + id));
         return bookMapper.toDto(book);
     }
+    
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    @Override
+    public BookDto updateById(Long id, BookRequestDto bookDto) {
+        Book book = bookMapper.toModel(bookDto);
+        return bookMapper.toDto(bookRepository.save(book));
```

Comment:
What is the difference between this method and a create method so far? how will it know that we should **update** and not create a new book? id that you receive is not even used anywhere